code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -90,6 +90,24 @@
 	P.amount = 5
 	P.loc = get_turf(holder.my_atom)
 
+/datum/chemical_reaction/slimeglass
+	name = "Slime Glass"
+	id = "m_glass"
+	result = null
+	required_reagents = list("water" = 1)
+	result_amount = 1
+	required_container = /obj/item/slime_extract/metal
+	required_other = 1
+
+/datum/chemical_reaction/slimeglass/on_reaction(datum/reagents/holder)
+	feedback_add_details("slime_cores_used","[type]")
+	var/obj/item/stack/sheet/metal/M = new /obj/item/stack/sheet/glass
```

Comment:
You're typecasting a sheet of glass as a metal sheet, you need to typecast the new object with the right type.
e.g. : `var/obj/item/stack/sheet/glass/G = new /obj/item/stack/sheet/glass`
But since you typecast your var you can simply drop the "/obj/item/stack/sheet/glass" after `new` since the type of the stuff you want to create is implied by the typecasting. So you get: `var/obj/item/stack/sheet/glass/G = new`